src/main/java/com/puppycrawl/tools/checkstyle/GlobalStatefulCheck.java

```diff
@@ -38,7 +38,8 @@
  * This is similar to multi-file validation, which checkstyle does not support fully yet.
  * Please refer to https://github.com/checkstyle/checkstyle/issues/3540 for details.
  *
- * @noinspection AnnotationClass, ClassIndependentOfModule, unused, ClassOnlyUsedInOnePackage
+ * @noinspection AnnotationClass, ClassIndependentOfModule, ClassOnlyUsedInOnePackage
+ * @noinspectionreason we support only Java 11+, we keep this annotation at top level by design
```

Comment:
How do you feel about a new, separate `@noinspectionreason` for each noinspection item? I feel this may make it easier to maintain multiples instead of redoing existing lines to add a comma or such?

Reply:
Makes sense, good idea.

Reply:
Done.